addons/mail/static/src/core/public_web/store_service_patch.js

```diff
@@ -14,13 +14,8 @@ patch(Store.prototype, {
         this.discuss = { activeTab: "main" };
         this.env.bus.addEventListener(
             "discuss.channel/new_message",
-            ({ detail: { channel, message } }) => {
-                if (this.env.services.ui.isSmall || message.isSelfAuthored) {
-                    return;
-                }
-                if (channel.isCorrespondentOdooBot && this.odoobotOnboarding) {
-                    // this cancels odoobot onboarding auto-opening of chat window
-                    this.odoobotOnboarding = false;
```

Comment:
Note: this was just not a good idea, removed all odoobot execeptions.
Added `silent` which is only enabled for onboarding specifically.
Other potentially important messages from OdooBot are no longer ignored.